service/history/workflow/task_refresher.go

```diff
@@ -199,6 +191,15 @@ func (r *TaskRefresherImpl) refreshTasksForWorkflowStart(
 	}
 
 	return nil
+
+}
+
+func (r *TaskRefresherImpl) refreshTasksForWorkflowStart(
```

Comment:
Why do we need this function?

Reply:
not needed anymore, remove in the next rev